restful.module

```diff
@@ -451,7 +451,7 @@ function _restful_get_data_from_menu_status(&$var) {
 
     case MENU_ACCESS_DENIED:
       $class_name = '\Drupal\restful\Exception\ForbiddenException';
-      $message = 'Access denied.';
+      $message = 'Access denied while checking menu access.';
```

Comment:
I wouldn't change it. "menu access" is very Drupal. It's simply access denied ;)